### DISCUSSION TITLE
Download: Purchased vs Subscribed?

### DISCUSSION THREAD

**Discussion by ObiSandwich** [heliomarpm/udemy-downloader-gui#103] [Q&A] [answered]
Hello... To download a course via this app, do I **Purchase** a course or does the app work with a paid **Subscription** on any course that I have subscribed to?

Thanks

**Comment by heliomarpm** [comment, marked as answer]
> Hello... To download a course via this app, do I **Purchase** a course or does the app work with a paid **Subscription** on any course that I have subscribed to?
> 
> Thanks

Hi, this app will facilitate the download of the courses you are enrolled in Udemy.  Note that some courses are protected by DRM encryption, which will prevent the video from downloading.

**Comment by ObiSandwich** [comment]
Thanks for the quick response :) Looking to set it up tonight... How do I identify the DRM protected courses?

**Reply by heliomarpm** [reply]
> Thanks for the quick response :) Looking to set it up tonight... How do I identify the DRM protected courses?

You will be alerted the moment the download finds some encrypted video and it will stop the download.
There is even an option to ignore encrypted videos, this option is useful when you want to continue downloading attachments.

**Comment by ObiSandwich** [comment]
Thank you

**Comment by heliomarpm** [comment]
functionality available in version 1.12
